site/dat/wiki/RequirementsForURLContent.wiki

```diff
@@ -0,0 +1,65 @@
+= Requirements For URL Content =
```

Comment:
The file name should be different. Something like `PluginRepositoryDescriptorFormat.wiki`